golang/examples/memcached.go

```diff
@@ -0,0 +1,40 @@
+package examples
+
+import (
+  "fmt"
+  psh "github.com/platformsh/config-reader-go/v2"
+  "github.com/bradfitz/gomemcache/memcache"
+)
+
+func FormattedCredentialsMemcached(creds psh.Credential) (string, error) {
+  formatted := fmt.Sprintf("%s:%d", creds.Host, creds.Port)
+  return formatted, nil
+}
```

Comment:
Unnecessary once the formatters are committed to the config reader, I presume?

Reply:
Yes, this was just making them work before config-reader got merged